setup.py

```diff
@@ -27,10 +27,10 @@
     "djangorestframework>=3.11.1,<4.0",
     "django-filter>=2.2,<23",
     "draftjs_exporter>=2.1.5,<3.0",
-    "Pillow>=9.1.0,<10.0.0",
+    "Pillow>=6.0.0,<11.0.0",  # update together with willow
```

Comment:
Did you intend to reduce the lower bound here? This was set as 9.1.0 in #10911.

Reply:
I did not. Copy/pasta error. Fixed

Reply:
I do wonder whether we need the Pillow line at all, since at the end of the day we are restricted by the Pillow versions that Willow supports 🤔

Reply:
^ I've been wondering about that as well. If that's the case it's going to be so much easier to handle any security fixes coming from Pillow as we can just make a new release of Willow without a new release of Wagtail.

Reply:
Went back and checked, and indeed we don't require either Pillow or Wand in Willow - https://github.com/wagtail/Willow/blob/main/pyproject.toml#L28-L31 So this was but a dream

Reply:
Maybe the answer is to add to the `[project.optional-dependencies]` in Willow, so that Wagtail requires `Willow[Pillow,heif]>=1.6.3,<1.7` and Willow specifies `Pillow>=9.1.0,<11.0.0` in that optional dependency. There would need to be a non-heif option too. Then Wagtail can remove the direct dependency on Pillow.

I'm not sure if this would introduce any breaking changes, especially to non-Wagtail uses of Willow.

Reply:
That is actually a pretty good suggestion

Reply:
https://github.com/wagtail/Willow/pull/130